error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/ReactorTemplates.java

```diff
@@ -271,6 +272,43 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /**
+   * Prefer {@link Flux#concatMapIterable(Function)} over {@link Flux#concatMap(Function)} with
+   * {@link Flux#fromIterable(Iterable)}.
+   */
+  static final class FluxConcatMapFromIterable<S> {
+    @BeforeTemplate
+    Flux<S> before(Flux<? extends Iterable<S>> flux) {
+      return Refaster.anyOf(
+          flux.concatMap(list -> Flux.fromIterable(list)), flux.concatMap(Flux::fromIterable));
+    }
+
+    @AfterTemplate
+    @UseImportPolicy(STATIC_IMPORT_ALWAYS)
+    Flux<S> after(Flux<? extends Iterable<S>> flux) {
+      return flux.concatMapIterable(identity());
+    }
+  }
+
+  /**
+   * Prefer {@link Flux#flatMapIterable(Function, int)} over {@link Flux#flatMap(Function, int)}
+   * with {@link Flux#fromIterable(Iterable)}.
+   */
+  static final class FluxFlatMapFromIterable<S> {
+    @BeforeTemplate
+    Flux<S> before(Flux<? extends Iterable<S>> flux, int concurrency) {
+      return Refaster.anyOf(
+          flux.flatMap(list -> Flux.fromIterable(list), concurrency),
+          flux.flatMap(Flux::fromIterable, concurrency));
```

Comment:
Hum, I guess that if `concurrency` is `1`, we want to rewrite that to `concatMapIterable` to be consistent with rules above 🤔 
So maybe I'll add a new entry to `FluxConcatMapFromIterable` above that rewrites specifically `flatMap(..., 1)` 🤔 
Is there such a thing as priority of rules? 
As this actually conflicts with `FluxConcatMap`...